src/header/template.njk

```diff
@@ -0,0 +1,61 @@
+{% from "phase-banner/macro.njk" import govukPhaseBanner %}
+
+<header class="govuk-header {{ params.classes if params.classes }}" role="banner" data-module="header"
+        {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+  <div class="govuk-header__container {{ params.containerClasses if params.containerClasses }}">
+
+    <div class="govuk-header__logo">
+      <a href="{{ params.homepageUrl }}" class="govuk-header__link govuk-header__link--homepage">
```

Comment:
could we consider having a default of / here?

Reply:
Updated, added `/` as default value.

Reply:
👍